src/client/pages/MaintenancePage.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { MainLayout } from '@/client/layouts/Main';
 import { MainBodyText } from '@/client/components/MainBodyText';
+import { MinimalLayout } from '@/client/layouts/MinimalLayout';
 
 export const MaintenancePage = () => (
-	<MainLayout pageHeader="We’ll be back soon">
+	<MinimalLayout pageHeader="We’ll be back soon">
 		<MainBodyText>
 			Sorry for the inconvenience. We are currently performing some essential
 			maintenance. Please try again later.
 		</MainBodyText>
```

Comment:
Other places are using the new `leadText` prop for similar things?

Should this use `leadText`?